src/index_spec.js

```diff
@@ -0,0 +1,134 @@
+const sinon = require('sinon');
+const Uploader = require('./index.js');
+const FileAPI = require('file-api')
+const {expect} = require('chai');
+const {describe} = require('mocha');
+
+global.FormData = require('form-data');
+global.XMLHttpRequest = sinon.useFakeXMLHttpRequest();
+
+describe('Uploader', () => {
+    let uploader, file, fileList;
+
+    beforeEach(() => {
+        file = new FileAPI.File({ 
```

Comment:
I would add multiple files with different settings as an array, and run all tests over each element in the array.

Reply:
why?

Reply:
So you can test different cases.